lte/gateway/python/magma/pipelined/app/classifier.py

```diff
@@ -159,6 +166,11 @@ def _install_internal_pkt_fwd_flow(self):
                        priority=flows.MINIMUM_PRIORITY,
                        goto_table=self.config.internal_sampling_fwd_tbl)
 
+    def _install_internal_conntrack_flow(self):
+        match = MagmaMatch(in_port=self.config.internal_conntrack_port)
+        flows.add_flow(self._datapath,self.tbl_num, match,
```

Comment:
We have a flow. resubmit to next service function can we use that instead?

Reply:
add_resubmit_next_service_flow

Reply:
Yes. Changes done.